backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/sectionheader/internal/BpkSectionHeaderImpl.kt

```diff
@@ -80,14 +81,14 @@ fun BpkSectionHeaderImpl(
                 if (isTablet) {
                     BpkButton(
                         text = it.text,
-                        onClick = { it.onClick.invoke() },
+                        onClick = { it.onClick },
```

Comment:
i think you still need the brackets (`onClick()`) OR you can just set `onClick = it.onClick`

Reply:
ops, fixed ✅